mybinder/values.yaml

```
@@ -42,7 +42,7 @@ tags: {}
 
 etcJupyter:
   jupyter_notebook_config.json:
```

Also need the right filename, I think:

```suggestion
  jupyter_server_config.json:
```